lesson_5/difficultCycles.js

```diff
@@ -0,0 +1,83 @@
+// Вивести числа від 20 до 30 через пропуск, використовуючи крок 0,5 (20 20,5 21 21,5….).
+const result = [];
+
+for (let i = 20; i < 31;) {
+  result.push(i);
+  i = i + 0.5;
+}
+console.log(result);
```

Comment:
This is incorrect solution
the result have to be 
`'20 20,5 21 21,5....'`
your result 
`[20, 20,5, 21 ...]`

Reply:
ok